std-bits/table/src/main/java/org/enso/table/data/column/operation/unary/UnaryRoundOperation.java

```diff
@@ -36,29 +63,34 @@ public boolean canApply(ColumnStorage storage) {
   @Override
   public ColumnStorage apply(
       ColumnStorage storage, MapOperationProblemAggregator problemAggregator) {
-    if (storage instanceof ColumnLongStorage longStorage) {
-      // For a long storage, the operation is an identity operation.
-      return longStorage;
+    if (storage instanceof ColumnLongStorage || storage instanceof BigIntegerStorage) {
+      // For an integral type storage, the operation is an identity operation.
+      return storage;
     }
 
     return super.apply(storage, problemAggregator);
   }
 
   @Override
   protected void applyObjectRow(
-      Object value, LongBuilder builder, MapOperationProblemAggregator problemAggregator) {
+      Object value, Builder builder, MapOperationProblemAggregator problemAggregator) {
```

Comment:
given we know the builder is an `InferredIntegerBuilder`, we could add an appendLong method which avoid the boxing.
(and yes we should test performance of the whole boxing game).

Reply:
Given that this implementation is benchmarking well, I would rather not add yet another specialization path to this, since we're expecting to refactor it soon.